pkg/controllers/state/suite_test.go

```diff
@@ -1121,17 +1121,25 @@ var _ = Describe("Cluster State Sync", func() {
 			})
 			ExpectApplied(ctx, env.Client, node)
 			ExpectReconcileSucceeded(ctx, nodeController, client.ObjectKeyFromObject(node))
+			ExpectMetricGaugeValue("karpenter_state_nodes_count", float64(i+1), make(map[string]string))
 		}
+
 		Expect(cluster.Synced(ctx)).To(BeTrue())
+		ExpectMetricGaugeValue("karpenter_state_synced", 1.0, make(map[string]string))
+		ExpectMetricGaugeValue("karpenter_state_nodes_count", 1000.0, make(map[string]string))
```

Comment:
can't you just do nil for the last parameter on each of these?

Reply:
Good shout